theano/gpuarray/opt.py

```diff
@@ -2182,6 +2247,11 @@ def local_gpu_svd(op, context_name, inputs, outputs):
 # to avoid a circular dependency problem with dnn
 from .dnn import (local_abstractconv_cudnn, local_abstractconv_gw_cudnn,
                   local_abstractconv_gi_cudnn)     # noqa: 402
+
+conv_metaopt = ConvMetaOptimizer()
+abstractconv_groupopt.register('conv_metaopt', conv_metaopt, 0, 'conv_meta')
```

Comment:
small question, the number argument in register is used for the position right?
because while I was going through the code in the method `register` in `LocalGroupdb` I found that the number argument is being taken as a tag.